CMakeLists.txt

```diff
@@ -607,6 +607,7 @@ set(SOURCES
         db/version_edit_handler.cc
         db/version_set.cc
         db/wal_manager.cc
+        db/wal_version.cc
```

Comment:
@cheng-chang  As you pointed out in the PR's description, a WAL addition or deletion is db-wise operation, not a mutation to a specific `Version`. Therefore, I wonder whether `wal_version` is the most accurate term.

Reply:
`wal_manifest` is more appropriate, but the `manifest` concept is named `version` in the codebase, so I prefer to keep it as `wal_version` for now.

Reply:
How about `wal_mutation` or `wal_addition` and `wal_deletion` or something like that, avoiding `version`. I'll leave this up to you.

Reply:
I renamed it to wal_edit